recipes/kpop/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "kpop" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/PaoloRibeca/KPop/releases/download/v{{ version }}/KPop-{{ version }}-Linux_x86-64.tar.xz  # [linux]
+    sha256: 915eb6d4e0b41b4f6409b5fd97188193106b3300734c92131ab8ad0618051e4a  # [linux]
+  - url: https://github.com/PaoloRibeca/KPop/releases/download/v{{ version }}/KPop-{{ version }}-MacOS_x86-64.tar.xz  # [osx]
+    sha256: 06c15129e9ecea58092f5e4d5df136a0c227815f011f80f63ad9093d70cc46d3  # [osx]
+
+build:
+  run_exports:
+    {{ pin_subpackage(name, max_pin="x") }}
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  run: 
+    - biocamlib
+
+test:
+  commands:
+    - KPopCount -V
+    - KPopCountDB -V  
+    - KPopTwistDB -V
+
+about:
+  home: https://github.com/PaoloRibeca/KPop
+  license: GPL-3.0-only
+  license_file: LICENSE
+  summary: 'An assembly-free and scalable method for the comparative analysis of microbial genomes, based on full k-mer spectra and dataset-specific transformations.'
+  description: |
+    kpop is an assembly-free and scalable method for the comparative analysis of microbial genomes, 
+    based on full k-mer spectra and dataset-specific transformations. It is built using OCaml <https://ocaml.org> and R <https://www.r-project.org/>,
+    and is dependant on the BiOCamLib package <https://github.com/PaoloRibeca/BiOCamLib>.
+  dev_url: https://github.com/PaoloRibeca/KPop
+
+extra:
+  recipe-maintainers:
+    - PaoloRibeca
+    - ryanmorrison22
+  skip-lints:
+    - should_be_noarch_generic
```

Comment:
I see - as this is essentially a binary-only distribution, I'll delete the `skip-lints` section and mark the package as no-`noarch`

Reply:
...but this does not work, because the "sources" (actually, the binaries) for BiOCamLib are platform-dependent. So I'll revert to what @ryanmorrison22 had done